### PR TITLE
CHANGELOG: update changelog for 1.8.0 release

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <wenliang@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+[1.8.0] - 2022-05-09
+--------------------
+
+### Changes
+
+- Support playbooks which use `gather_facts: false`
```
Comment (tyll) [inline CHANGELOG.md:9]: this seems like a feature to me, too. Or a bug fix. Not a Change.

Comment (liangwen12year) [reply CHANGELOG.md]: Updated, thanks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+### New features
+
+- Add support for routing rules
```
[the remark this answers is not in the document]
Comment (liangwen12year) [reply CHANGELOG.md]: Updated.

**Review by ffmancera — APPROVED** [review]

**Comment by liangwen12year** [comment]
Since the most recent commits in the main branch, I need to update this patch.

**Review by ffmancera — APPROVED** [review]

**Pull request merged**